hpctestlib/apps/utils/__init__.py

```diff
@@ -0,0 +1,9 @@
+import re
+import reframe as rfm
+
+__all__ = ['define_reference']
+
+
+def define_reference(test: rfm.RegressionTest):
+    test.reference_value = test.references[test.benchmark][0]
+    test.reference_difference = test.references[test.benchmark][1]
```

Comment:
I see what you're trying to do here. But the test attributes that you're setting here are test-specific, which means that they should not be part of a utils-style library. The fact that you might use the same names in different tests, doesn't mean that this has the same meaning in all the tests you use this. In any case, with the recommendations I made above, you no longer need this code.